src/Routes/RosPage/RosPage.js

```diff
@@ -195,10 +232,21 @@ class RosPage extends React.Component {
             });
         }
 
+        if (deletedOSFilters.length > 0) {
+            const resetFiltersList = deletedOSFilters[0]?.chips.map((chip) =>{
```

Comment:
Can we keep the name as `resetOSFilterList` as it is related to OS filters?

Reply:
On line [#L188](https://github.com/RedHatInsights/ros-frontend/blob/main/src/Routes/RosPage/RosPage.js#L188), we're using `resetFiltersList` for `state` filter as well and this constant is being used within its scope. I think we are good here.